content/en/docs/setup/install/multicluster/single-network/remote/index.md

```diff
@@ -0,0 +1,159 @@
+            </td>
+            <td>Specifies a human-readable cluster name.</td>
+            <td><code>${CLUSTER_2}</code></td>
+        </tr>
+        <tr>
+            <td><code>network</code></td>
+            <td>Specifies a network ID as an arbitrary string. All clusters in your mesh
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshID</code></td>
+            <td>Specifies a mesh ID as an arbitrary string. All clusters in your mesh share
+            the same mesh ID.</td>
+            <td><code>${MESH}</code></td>
+        </tr>
+        <tr>
+            <td><code>remotePilotAddress</code></td>
+            <td>IP address of the Istio ingress gateway of `Cluster_1`. </td>
+            <td>${DISCOVERY_ADDRESS}</td>
+        </tr>
+    </tbody>
+</table>
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy Istio in `Cluster_2` with the following steps:
+
+1. Set the value of the `DISCOVERY_ADDRESS` environment variable to the IP
+   address of the Istio ingress gateway of `Cluster_1` with the following
+   command:
+
+{{< text bash >}}
+$ export DISCOVERY_ADDRESS=$(kubectl \
+  --context=${CTX_1} \
+  -n istio-system get svc istio-ingressgateway \
+  -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+{{< /text >}}
+
+1. To pass configuration values to the Istio operator for installation, you
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+{{< text bash >}}
+$ cat <<EOF> ${WORK_DIR}/${CLUSTER_2}/install.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: ${MESH}
+      multiCluster:
+        clusterName: ${CLUSTER_2}
+      network: ${NETWORK_1}
+      # Access the control plane discovery server via the ingress
+      remotePilotAddress: ${DISCOVERY_ADDRESS}
+EOF
+{{< /text >}}
+
+1. Install Istio on `Cluster_2` with the following command:
+
+{{< text bash >}}
+$ istioctl --context=${CTX_2} manifest apply -f \
+  ${WORK_DIR}/${CLUSTER_2}/install.yaml
+{{< /text >}}
+
+1. Verify that the control plane of `Cluster_2` is ready with the following
+   command:
+
+{{< text bash >}}
+$ kubectl --context=${CTX_2} -n istio-system get pod
+NAME                                    READY   STATUS    RESTARTS   AGE
+istiod-f756bbfc4-thkmk                  1/1     Running   0          136m
+prometheus-b54c6f66b-q8hbt              2/2     Running   0          136m
+{{< /text >}}
+
+1. After the status of all pods is `Running`, you can continue configuring
+   your deployment.
+
+## Configure endpoint discovery
+
+To enable cross-cluster load balancing in your mesh, configure endpoint
+discovery. This feature requires that clusters share secrets between them. If
+the shared secrets provide the [needed trust](#configure-trust), each
```

Comment:
The link is right here.

It seems that the "needed trust" from the CA certs isn't enough, and you still need to set a secret up so that the control plane in CLUSTER_1 can query the API server in CLUSTER_2.

Reply:
That step is on line 146, thanks.